bootstrap_test.go

```diff
@@ -121,6 +121,7 @@ func (s *testSessionSuite) TestBootstrapWithError(c *C) {
 	store := newStore(c, s.dbNameBootstrap)
```

Comment:
This is a special test, related to bootstrap, so it use newStore.

```
newStoreWithBootstrap = newStore + Bootstrap
Bootstrap = newDomain + initialize
newDomain = ddl + loadInfoSchema
initialize = create tables user, db, variable and so on
```

In brief, we can't use a barely store, it must be bootstrap before use.
Before this PR, that process is done when CreateSession is called  the first time, 
now we separate it to clean up the code.